sdk/storage/azure-storage-blob/azure/storage/blob/blob_client.py

```diff
@@ -1387,6 +1388,7 @@ def _start_copy_from_url_options(self, source_url, metadata=None, incremental_co
             dest_access_conditions = get_access_conditions(kwargs.pop('destination_lease', None))
             options['source_modified_access_conditions'] = source_mod_conditions
             options['lease_access_conditions'] = dest_access_conditions
+            options['tier'] = tier.value if tier else None
```

Comment:
Thanks for fixing this! Also in a clean way! Just noticed it failed IC while I haven't pushed the fix.